applications/luci-app-babeld/root/usr/share/luci/menu.d/luci-app-babeld.json

```diff
@@ -0,0 +1,13 @@
+{
+	"admin/services/babeld": {
```

Comment:
I have mentioned this before but I don't think you understood me ( I remember that you asked about). I think the overview is better kept in the status menu tree for bable than in the configuration. You only show status information here, it is not configuration page. See this for example https://github.com/openwrt/luci/blob/be49059e4d4ce156cfdf1b0240b05072b495d882/applications/luci-app-wireguard/root/usr/share/luci/menu.d/luci-app-wireguard.json#L2